--- a/PKG-INFO
+++ b/PKG-INFO
@@ -22,7 +22,7 @@
 Requires-Dist: cachetools (>=4.2.1,<5.0.0); extra == "e2e"
 Requires-Dist: dataclasses (>=0.7,<0.8); python_version >= "3.6" and python_version < "3.7"
 Requires-Dist: future (>=0.18.2,<0.19.0)
-Requires-Dist: h11 (>=0.12.0,<0.13.0)
+Requires-Dist: h11 (>=0.12.0,<0.14.0)
 Requires-Dist: h2 (>=4.0.0,<5.0.0)
 Requires-Dist: jsonschema (>=4.4.0,<5.0.0)
 Requires-Dist: logbook (>=1.5.3,<2.0.0)
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@
  'aiohttp-socks>=0.7.0,<0.8.0',
  'aiohttp>=3.7.4,<4.0.0',
  'future>=0.18.2,<0.19.0',
- 'h11>=0.12.0,<0.13.0',
+ 'h11>=0.12.0,<0.14.0',
  'h2>=4.0.0,<5.0.0',
  'jsonschema>=4.4.0,<5.0.0',
  'logbook>=1.5.3,<2.0.0',
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -21,7 +21,7 @@
 aiohttp = "^3.7.4"
 aiofiles = "^0.6.0"
 dataclasses = { version = "^0.7", python = ">= 3.6, <3.7" }
-h11 = "^0.12.0"
+h11 = "^0.13.0"
 h2 = "^4.0.0"
 logbook = "^1.5.3"
 jsonschema = "^4.4.0"

